backend_plugin/src/main/evaluators/CorrectIndentationEvaluator.java

```diff
@@ -0,0 +1,131 @@
+package main.evaluators;
+
+import org.eclipse.jface.text.BadLocationException;
+import org.eclipse.jface.text.DocumentEvent;
+import org.eclipse.jface.text.IDocument;
+
+/**
+ * Evaluates DocumentEvent changes to determine if the user is changing the indentation of multiple sequential lines of code. If so,
+ * then the user should be notified of the auto indentation feature in Eclipse.
+ */
+public class CorrectIndentationEvaluator extends FeatureEvaluator {
+    private String lineBeforeChange;
+    private boolean whiteSpaceAddedOrRemoved;
+    private int lastIndentChangedLine;
+    private long lastCommentedLineTimeStamp;
```

Comment:
Should this be "lastIndentChangeTimeStamp" instead?

Reply:
Good catch